testing/it/ui-js/src/content/jcr_root/content/dam/core-components/contentfragments-tests/_jcr_content/folderThumbnail.dir/.content.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:dam="http://www.day.com/dam/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:file">
+    <jcr:content
+        dam:folderThumbnailPaths="[/content/dam/core-components-examples/library/sample-assets/snowy-mountain-glacier.jpg,/content/dam/core-components-examples/library/sample-assets/lava-into-ocean.jpg,/content/dam/core-components-examples/library/sample-assets/lava-rock-formation.jpg]"
```

Comment:
I don't think we need to add this folderThumbnail at all for the test content, plus the references are from the library